tests/Features/Expect/toContainEquals.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use PHPUnit\Framework\ExpectationFailedException;
+
+
+test('passes arrays', function () {
+    expect([1, 2, 42])->toContain('42');
+});
+
+test('passes arrays with multiple needles', function () {
+    expect([1, 2, 42])->toContain('42', '2');
+});
+
+test('failures', function () {
+    expect([1, 2, 42])->toContain('3');
+})->throws(ExpectationFailedException::class);
```

Comment:
I believe your goal with this was to prove that `toContain` does not analyze type and value, right? I don't know if we should keep these tests here because they are not part of the `toContainEquals` concept.

@nunomaduro , thoughts?

Reply:
Actually these should use `toContainEquals`, good catch.

Reply:
> Actually these should use `toContainEquals`, good catch.

If this is wrong, please update the PR.